recipes/7zip/19.00/conanfile.py

```diff
@@ -1,8 +1,10 @@
-from conans import ConanFile, tools, AutoToolsBuildEnvironment
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import download, chdir, replace_in_file
+from conans import tools, AutoToolsBuildEnvironment
 import os
 
-required_conan_version = ">=1.33.0"
+required_conan_version = ">=1.52.0"
```

Comment:
```suggestion
required_conan_version = ">=1.47.0"
```
I hate to be boring, but some users already complained about recipes asking the latest version. As this recipe does not need the latest version ...